alarms/models/circ_trans_alarms.py

```diff
@@ -42,14 +42,14 @@ def run_checks(self):
                 redshift_count = self.get_record_count(
                     self.redshift_client, redshift_query
                 )
-                if sierra_count != redshift_count:
-                    mismatch_log = build_redshift_mismatch_log(database_type='Sierra circ trans', 
-                                                               redshift_table=redshift_table,
-                                                               database_count=sierra_count,
-                                                               redshift_count=redshift_count) 
-                    self.logger.error(mismatch_log)
+                redshift_mismatch_alarm(logger=self.logger,
+                                            database_type='Sierra circ trans', 
```

Comment:
nit: weird line spacing and prefer double quotes to single (here and elsewhere) -- `black` should autoformat